### PR TITLE
add att.normalized to hold @orig and @norm, and documentation to DI and AI

### PR DESCRIPTION
**Pull request by bansp**
Please see ticket #1973 for a description of this pull request

### CODE REVIEW

**Comment by bansp** [comment]
(Rebased this branch to clean up the commit mess -- a little positive side-effect of the Corona fing...)

File: P5/Source/Guidelines/en/AI-AnalyticMechanisms.xml [hunk 1]
```diff
@@ -481,6 +493,7 @@ This matter is taken up in detail in section <ptr target="#AILA"/>.
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/pc.xml"/>
 </specGrp>
 <specGrp xml:id="DAILA" n="Linguistic Word-level Attributes">
+<include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/att.normalized.xml"/>
```
Comment (tuurma) [inline P5/Source/Guidelines/en/AI-AnalyticMechanisms.xml:496]: After council f2f discussion we'd suggest to name the class att.lexicographic.normalized to make it very clear that these attributes are only designed for a very particular purpose.

File: P5/Source/Guidelines/en/DI-PrintDictionaries.xml [hunk 2]
```diff
@@ -2691,8 +2691,9 @@ captured by encoding one <soCalled>dominant</soCalled> view in the character
 data content of the document, and encoding the other using attribute values on
 the appropriate elements. If all tags were to be removed, the remaining
 characters would be those of the dominant view of the text. </p>
-<p>The attribute class <ident type="class">att.lexicographic</ident> is used to
-provide attributes for use in encoding multiple views of the same dictionary
+<p>The attribute class <ident type="class">att.lexicographic</ident> (which 
+includes the attributes <att>norm</att> and <att>org</att> from class <ident type="class">att.normalized</ident>)
```
Comment (tuurma) [inline P5/Source/Guidelines/en/DI-PrintDictionaries.xml:2695]: att.lexicographic.normalized

File: P5/Source/Specs/att.lexicographic.xml [hunk 3]
```diff
@@ -14,6 +14,7 @@ $Id$
   <desc versionDate="2007-01-21" xml:lang="it">definisce un insieme di attributi globali disponibili per gli elementi dell'insieme base di marcatori per i dizionari.</desc>
   <classes>
     <memberOf key="att.datcat"/>
+    <memberOf key="att.normalized"/>
```
Comment (tuurma) [inline P5/Source/Specs/att.lexicographic.xml:17]: att.lexicographic.normalized

File: P5/Source/Specs/att.linguistic.xml [hunk 4]
```diff
@@ -7,7 +7,9 @@ $Id$
 --><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
 <classSpec xmlns="http://www.tei-c.org/ns/1.0" module="analysis" xml:id="ATTLING" type="atts" ident="att.linguistic">
   <desc versionDate="2017-03-24" xml:lang="en">provides a set of attributes concerning linguistic features of tokens, for usage within token-level elements, specifically <gi>w</gi> and <gi>pc</gi> in the analysis module.</desc>
-  <classes/>
+  <classes>
+    <memberOf key="att.normalized"/>
```
Comment (tuurma) [inline P5/Source/Specs/att.linguistic.xml:11]: att.lexicographic.normalized

File: P5/Source/Specs/att.normalized.xml [hunk 5]
```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="analysis" xml:id="ATTNORM" type="atts" ident="att.normalized">
```
Comment (tuurma) [inline P5/Source/Specs/att.normalized.xml:8]: ident="att.lexicographic.normalized"

Likewise, rename the file accordingly

File: P5/Source/Specs/att.normalized.xml [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+        <p>An example from the EarlyPrint project showing the use of both <att>norm</att> and <att>orig</att>. The <att>orig</att> attribute preserves the original version (sometimes with spelling errors, often with printer abbreviations), the element content resolves printer abbreviations but retains the original orthography, and the <att>norm</att> attribute holds normalized values:</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <w lemma="commandment" pos="n1" norm="commandment" xml:id="b9avr-018-a-7720" orig="commandemēt">commandement</w>
+        </egXML>
+      </exemplum>
+    </attDef>
+
+  </attList>
+  <remarks versionDate="2020-02-12" xml:lang="en">
+    <p>It needs to be stressed that the two attributes in this class are meant for strictly
```
Comment (tuurma) [inline P5/Source/Specs/att.normalized.xml:134]: Yes!

**Comment by bansp** [comment]
Will do that today, thanks.

**Comment by tuurma** [comment]
@peterstadler @sydb I believe this PR is now good to go, would you like to do the honors? Thanks @bansp for your submission!

**Pull request merged**